### ISSUE TITLE
 Uninstrument Sim, and move the simStarted event and the ScreenShot method to phetio (PhETIO)

### ISSUE THREAD

**Issue by zepumph** [phetsims/joist#481]
From https://github.com/phetsims/faradays-law/issues/107, this will simplify the implementation.

**Comment by zepumph** [comment]
I uninstrumented `Sim.js` and deleted `SimIO.js`. I basically did three things:

1. Moved the `getScreenshotDataURL` to `PhETIO.js`
2. Converted `simStartedEvent` to be run from the `phetio.js` instance (but still in sim.js)
3. We couldn't actually uninstrument Sim.js, because it holds the root tandem, so I made it an ObjectIO (maybe it could be a SimIO.js but have SimIO.js not do anything)
4. Fixed a small bug where simStartedMetadata from the wrapper wasn't being added into the simStarted event. This is probably from phetsims/faradays-law#107.

@samreid please review. It's a bit weird to have the root tandem now just be an ObjectIO, but maybe that's because it's new to me.

**Comment by samreid** [comment]
> We couldn't actually uninstrument Sim.js, because it holds the root tandem, so I made it an ObjectIO (maybe it could be a SimIO.js but have SimIO.js not do anything)

I didn't understand this part.  Why does "holding the root tandem" imply that it must be instrumented?

**Comment by zepumph** [comment]
I though that it was important to see the root tandem as an instance that says "Represents the entire simulation."  

Maybe this isn't important at all though, and we can just uninstrument sim such that there is no "instance" with the phetioID of the root tandem (sim name).

**Comment by zepumph** [comment]
I removed the sim instrumentation. Now all the phetio code in Sim.js are properties that grab tandem directly from the root tandem, and code in the `initializePhetio` function, only called if phetio is enabled.

I tested these changes by:
1. Phet fuzz 
2. Phet-io fuzz
3. wrapper tests
4. studio for FL to confirm that sim is uninstrumented.

@samreid back to you for a spot check and then close if all is good to you. Thanks!
